[2023-03-01 12:32:29,536] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_age 2022-10-26T21:05:00+00:00 [queued]>
[2023-03-01 12:32:29,683] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: with_python_operator.get_age 2022-10-26T21:05:00+00:00 [queued]>
[2023-03-01 12:32:29,688] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:32:29,689] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2023-03-01 12:32:29,691] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2023-03-01 12:32:29,855] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): get_age> on 2022-10-26T21:05:00+00:00
[2023-03-01 12:32:29,894] {standard_task_runner.py:52} INFO - Started process 8085 to run task
[2023-03-01 12:32:30,020] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'with_python_operator', 'get_age', '2022-10-26T21:05:00+00:00', '--job-id', '1814', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/02_python_operator.py', '--cfg-path', '/tmp/tmplm9e3ew2', '--error-file', '/tmp/tmpl_sia1oj']
[2023-03-01 12:32:30,053] {standard_task_runner.py:77} INFO - Job 1814: Subtask get_age
[2023-03-01 12:32:30,710] {logging_mixin.py:104} INFO - Running <TaskInstance: with_python_operator.get_age 2022-10-26T21:05:00+00:00 [running]> on host abe4c4997ef7
[2023-03-01 12:32:31,303] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataheroes
AIRFLOW_CTX_DAG_ID=with_python_operator
AIRFLOW_CTX_TASK_ID=get_age
AIRFLOW_CTX_EXECUTION_DATE=2022-10-26T21:05:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-26T21:05:00+00:00
[2023-03-01 12:32:31,400] {python.py:118} INFO - Done. Returned value was: None
[2023-03-01 12:32:31,499] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=with_python_operator, task_id=get_age, execution_date=20221026T210500, start_date=20230301T123229, end_date=20230301T123231
[2023-03-01 12:32:32,398] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-03-01 12:32:32,591] {local_task_job.py:146} INFO - Task exited with return code 0
